packages/orm/entity/TaskOrder.ts

```diff
@@ -20,10 +21,10 @@ export class TaskOrder extends BaseEntity {
   @Column({ type: "uuid", comment: "S3 object key of task order pdf" })
   fileId: string;
 
-  @Column({ length: 256, comment: "name of pdf file when uploaded" })
+  @Column({ type: "varchar", length: 256, comment: "name of pdf file when uploaded" })
   fileName: string;
 
-  @Column({ nullable: true, comment: "pdf file size in bytes" })
+  @Column({ type: "integer", nullable: true, comment: "pdf file size in bytes" })
```

Comment:
Is this informing TypeORM of the DB structure? Or do we need to write a migration for this as well?

Reply:
Good question.  Had been encountering `ColumnTypeUndefinedError` until adding these.  This is manually specifying the column type to TypeORM.  Attempts to get column type automatically must have been failing in these cases.

You had me concerned that we might need a migration, however, I checked each column changed against the migration file and found the expected type for each within the CREATE TABLE statements.  So I think we're ok.